docs/src/proxy.md

```diff
@@ -41,6 +41,11 @@ A session represents ongoing communication flow between a client and an [Upstrea
 
 The proxy exposes the following general metrics (See the metrics sub-sections for metrics specific to other Quilkin components, e.g for metrics related to packet flow see [sessions metrics][session-metrics], or metrics exported by individual filters can be found in the documentation for each filter):
 
+- `quilkin_proxy_processing_time_seconds{event}` (Histogram)
+The total duration of time in seconds that it took to process a packet.
+  * The `event` label is either `read` or `write` which corresponds to either `on_read` or `on_write` execution within 
```

Comment:
> corresponds to either `on_read` or `on_write` execution within the proxy.


this reads a bit confusing, I don't think we use `on_read`/`on_write` in our docs/code but rather `read`/`write` if this is referring to filters? If this isn't filters specific but on the proxy level we should probably instead use the upstream/downstream terminology we have since read/write is filter specific